external/fv3fit/fv3fit/_shared/utils.py

```diff
@@ -8,3 +18,107 @@ def parse_data_path(data_path: Union[List, str]):
         return data_path[0]
     else:
         return data_path
+
+
+class StackedBatches(Sequence[xr.Dataset]):
```

Comment:
Please put these in a new file called `stacking.py` or something similar, to avoid putting it in a `utils` file.